lib/mixlib/shellout.rb

```diff
@@ -297,7 +289,6 @@ def parse_options(opts)
           self.password = setting
         when 'user'
           self.user = setting
-          self.with_logon = setting
```

Comment:
Yeah restore with_login and the entire variable, and wire it back up to the windows logic the way it was.  We most likely depend on all that behavior somewhere.